### ISSUE TITLE
amplify console undocumented in cli for a reason?

### ISSUE THREAD

**Issue by phishy** [aws-amplify/docs#715]
Running `amplify` or `amplify api` does not produce the `console` command in cli help. 

Is this intentional? I had only known of its existence by stumbling upon it in the docs somewhere.

**Comment by phishy** [comment]
Why was this moved to docs? I was curious if this was a pull request I could handle for the cli repo. First, I wanted to check to see if it was intentional.

**Comment by kaustavghosh06** [comment]
@phishy No, it isn't intentional. It should be a part of the docs + cli. Would be great, if you could submit a PR for the same.

**Comment by mlabieniec** [comment]
@kaustavghosh06 it looks like what @phishy is saying is that `amplify api` does not support/have the `console` command. Not all category commands implement the console command, so this should be in aws-amplify/amplify-cli as a feature-request.
